src/main/java/course_project/Utils/ConsoleHandling/Printer.java

```diff
@@ -0,0 +1,184 @@
+package course_project.Utils.ConsoleHandling;
+
+import ConsoleColors.ConsoleColors;
+import course_project.components.Field;
+import course_project.components.Ship;
+import course_project.enums.ShipType;
+
+public class Printer {
+
+
+    public void printGreeting() {
+        String GREETING = "Welcome to Sea Battle, stranger!";
+        System.out.println(GREETING);
+    }
+
+    public void printAskHowFill() {
+        String ASK_HOW_TO_FILL = "How Would you like to place your ships? " +
+                "Please, enter \" anything \" to place them manually or just press enter to place them automatically \n" +
+                ConsoleColors.RED + "Bear in mind, that any input will be counted like manual mode!" + ConsoleColors.RESET;
+        System.out.println(ASK_HOW_TO_FILL);
+    }
+
+    public void printHeader() {
+        String HEADER = "    A  B  C  D  E  F  G  H  I  J ";
+
+        System.out.println(HEADER);
+    }
+
+    public void printYouMiss() {
+        String YOU_MISS = "Oh no, you missed  :(";
+        System.out.println(YOU_MISS);
+    }
+
+    public void printPCMiss() {
+        String AI_MISSED = "Computer missed, thank goodness!";
+        System.out.println(AI_MISSED);
+    }
+
+    public void printYouHit() {
+        String YOU_HIT = "What a shoot, you hit it!";
+        System.out.println(YOU_HIT);
+    }
+
+    public void printYouWasHit() {
+        String YOU_WAS_HIT = "Uh, oh, your ship is falling apart!";
+        System.out.println(YOU_WAS_HIT);
+    }
```

Comment:
Opt: methdos like that are not needed, instead you should introduce static constants and just call sout(constant) where needed